include/swift/Option/Options.td

```diff
@@ -466,4 +466,8 @@ def sanitize_EQ : CommaJoined<["-"], "sanitize=">,
   Flags<[FrontendOption, NoInteractiveOption]>, MetaVarName<"<check>">,
   HelpText<"Turn on runtime checks for erroneous behavior.">;
 
+def sanitize_coverage_EQ : Joined<["-"], "sanitize-coverage=">,
+  Flags<[FrontendOption, NoInteractiveOption]>,
+  MetaVarName<"<coverage_type>">, HelpText<"Specify sanitize coverage type">;
```

Comment:
Sure, that's a bit better. I guess command-line help text isn't supposed to be the full documentation anyway.